tests/test_wave_03.py

```diff
@@ -54,13 +54,10 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert ACTION_1 not in friends_unique_movies
+    assert INTRIGUE_1 not in friends_unique_movies
```

Comment:
One improvement here might be to check for what we actually expect to be in `friends_unique_movies`